docs/reference/cluster/get-desired-balance.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[get-desired-balance]]
+=== Get desired balance API
+++++
+<titleabbrev>Get desired balance</titleabbrev>
+++++
+
+NOTE: {cloud-only}
+
+Exposes the desired balance and basic metrics.
+
+[[get-desired-balance-request]]
+==== {api-request-title}
+
+[source,console]
+--------------------------------------------------
+GET /_internal/desired_balance
+--------------------------------------------------
+// TEST[skip:Can't reliably test desired balance]
+
+The API returns the following result:
+
+[source,console-result]
+--------------------------------------------------
+{
+	"stats": {
+		"computation_active": false,
+		"computation_submitted": 5,
+		"computation_executed": 5,
+		"computation_converged": 5,
+		"computation_iterations": 4,
+		"computation_converged_index": 4,
+		"computation_time_in_millis": 0,
+		"reconciliation_time_in_millis": 0
+	},
+	"routing_table": {
+		"test": {
+			"0": {
+				"current": {
+					"state": "STARTED",
+					"primary": true,
+					"node": "UPYt8VwWTt-IADAEbqpLxA",
+					"node_is_desired": false,
```

Comment:
Not sure if it is just a sample or implementation issue as well, but this should be true as this value is contained in `node_ids` in `desired` object below. Anyways I would suggest to change to true if this is presented as a doc

Reply:
Good call!